### PR TITLE
Add proto3 install scripts to Dockerfiles and update docs.

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -59,6 +59,19 @@ We recommend using `wstool <http://wiki.ros.org/wstool>`_ and `rosdep
...
+    git checkout tags/v3.4.1
+    mkdir build
+    cd build
+    cmake -G Ninja \
+      -DCMAKE_BUILD_TYPE=Release \
+      -Dprotobuf_BUILD_TESTS=OFF \
+      ../cmake
+    ninja
+    sudo ninja install
+
```
Comment (SirVer) [inline docs/source/index.rst]: As discussed, rather refer to the installation script in Cartographer.

Comment (cschuet) [reply docs/source/index.rst]: Done

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**